java/values.yaml

```diff
@@ -16,3 +16,4 @@ livenessDelay: 30
 livenessTimeout: 3
 livenessPeriod: 15
 livenessFailureThreshold: 3
+azureApplicationInsightsInstrumentKey: '00000000-0000-0000-0000-000000000000'
```

Comment:
what about
```suggestion
applicationInsightsInstrumentationKey: '00000000-0000-0000-0000-000000000000'
```

Reply:
Sure, will change.  keeping it small will be helpful